### ISSUE TITLE
Sliders for adjusting the Min/Max dB of the Waterfall display instead of entering dB values

### ISSUE THREAD

**Issue by reflectingme** [ExpertSDR3/ExpertSDR3-SUGGESTIONS#125]
I consider if would be more useful if there were sliders that allowed the adjustment of the Max dB and Min dB values instead of having to manually type in dB figures into those fields.  It would also help if any changes made to a slider were reflected in the waterfall display as the adjustments were being made.  Right now, a value needs to be entered in the field **and** tabbed out or press enter to see the result of the change.

So perhaps instead of this...

<img width="360" alt="Screenshot 2022-11-07 at 09 27 29" src="https://user-images.githubusercontent.com/47534300/200277779-56785118-d6b4-4b1a-899d-fddf60447aa4.png">

We might see something such as this...

<img width="722" alt="Screenshot 2022-11-07 at 09 40 10" src="https://user-images.githubusercontent.com/47534300/200277860-c8bdc15c-4f8f-4f88-896b-71947fead954.png">

Would make it easier and quicker to adjust these values

**Comment by ZL3RC** [comment]
Actually this is possible now. 
Select the Min or Max field and then scroll up and down with your mouse or track pad to change the number.
Its the same for all the fields in the UI that I have need to change.

**Comment by reflectingme** [comment]
> Actually this is possible now.
> Select the Min or Max field and then scroll up and down with your mouse or track pad to change the number.
> Its the same for all the fields in the UI that I have need to change.

Yep, gotcha, and that works sort of OK'ish.  It still isn't sliders though which I feel would be a neater of adjusting the dB values and it would be consistent with the current GUI interface.
